Create Basic Mathematical Operations

Pandz18 opened the pull request:
Hey , this is a program where I have used all the basic mathematical operations by using functions . I hope this is good , if there should be any changes or anything that I can do to make it more efficient , please do let me know 

Cheers

Commit:
Create Basic Mathematical Operations

Hey , this is a program where I have used all the basic mathematical operations by using functions . I hope this is good , if there should be any changes or anything that I can do to make it more efficient , please do let me know 

Cheers

10xsai commented on Basic Mathematical Operations/C++Sol.cpp:
```diff
@@ -0,0 +1,130 @@
+
+void DIFF()
+{
+ if(number1<number2)
```
I think you can use abs( ) inbuilt function to directly find absolute difference instead of a for loop.

Pandz18 replied:
Oh yeah , makes sense , thank you 

will change that too

10xsai commented on Basic Mathematical Operations/C++Sol.cpp:
```diff
@@ -0,0 +1,130 @@
+void PROD()
+{
+	prod=number1*number2;
+	diff=number2-number1;
```
why difference here?

Pandz18 replied:
> why difference here?

Oh gosh , must've been an error , I might have been thinking ahead of my typing speed , will clear that up

Commit:
Update C++Sol.cpp

Pandz18 commented:
Hi , just an update , I have changed the two mistakes

10xsai commented on Basic Mathematical Operations/C++Sol.cpp:
```diff
@@ -0,0 +1,121 @@
+}
+
+void DIV()
+{
```
this code looks good but this is my suggestion
div_val = max(num1, num2)/min(num1, num2)
this take one operation more but more elegant.

Pandz18 replied:
Yeah , that makes it a lot easier

10xsai commented on Basic Mathematical Operations/C++Sol.cpp:
```diff
@@ -0,0 +1,121 @@
+}
+
+void PERM()
+{
```
My suggestion
you can create a factorial function and use it twice

Pandz18 replied:
Ohh...sure ok

Commit:
Update C++Sol.cpp

10xsai committed:
Update C++Sol.cpp

I think you might have confused with factorial part. So, I made some changes and this is what I was thinking. Did some code formatting.
